exonum/src/blockchain/config.rs

```diff
@@ -138,11 +141,11 @@ impl ConsensusConfig {
         const MIN_TXS_BLOCK_LIMIT: u32 = 100;
         const MAX_TXS_BLOCK_LIMIT: u32 = 10_000;
 
-        if self.round_timeout <= 2 * self.max_propose_timeout {
+        if self.first_round_timeout <= 2 * self.max_propose_timeout {
```

Comment:
I suppose some validation of `round_timeout_increase` should be added too. Do you intentionally allow zero value?